packages/drivers/odsp-driver/src/compactSnapshotWriter.ts

```diff
@@ -84,7 +84,7 @@ function writeBlobsSection(node: NodeCore, blobs: Map<string, IBlob | ArrayBuffe
  * @param node - node to serialize to.
  * @param ops - ops that is being serialized
 */
-function writeOpsSection(node: NodeCore, ops: ISequencedDeltaOpMessage[]) {
+function writeOpsSection(node: NodeCore, ops: ISequencedDeltaOpMessage[] | ISequencedDocumentMessage[]) {
```

Comment:
writer code is mostly for debugging, right? Do we need to support both formats? Why?